lib/tic_tac_toe_rules.rb

```diff
@@ -0,0 +1,40 @@
+class TicTacToeRules
+    def winner?(player_stats)
+        return false if player_stats.length <= 2 
+        evaluate_player_stats(player_stats)
+    end
+
+    def evaluate_player_stats(player_stats)
+        player_stats = player_stats.sort()
```

Comment:
I don't remember, but my tests do pass without it. I'll have to revisit that line. I think it was leftover before I started mapping through the stats. Since I'm mapping through the stats, there is no reason to sort. I'll delete that.